src/constructor.io/models/Catalog/ItemGroupsRequest.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Constructorio_NET.Utils;
+
+namespace Constructorio_NET.Models
+{
+    /// <summary>
+    /// Constructor.io Items Request Class.
```

Comment:
Minor issue here with the comment refering "Items Request" but other than this looks good to me.